src/search/landmarks/landmark_heuristic.cc

```diff
@@ -137,30 +115,17 @@ void LandmarkHeuristic::compute_landmark_graph(const plugins::Options &opts) {
 }
 
 void LandmarkHeuristic::generate_preferred_operators(
-    const State &state, ConstBitsetView &past) {
+    const State &state, ConstBitsetView &future) {
     /*
-      Find operators that achieve landmark leaves. If a simple landmark can be
-      achieved, prefer only operators that achieve simple landmarks. Otherwise,
-      prefer operators that achieve disjunctive landmarks, or don't prefer any
-      operators if no such landmarks exist at all.
-
+      Find operators that achieve future landmarks.
       TODO: Conjunctive landmarks are ignored in *lm_graph->get_node(...)*, so
        they are ignored when computing preferred operators. We consider this
        a bug and want to fix it in issue1072.
     */
     assert(successor_generator);
     vector<OperatorID> applicable_operators;
     successor_generator->generate_applicable_ops(state, applicable_operators);
-    vector<OperatorID> preferred_operators_simple;
-    vector<OperatorID> preferred_operators_disjunctive;
-
-    bool all_landmarks_past = true;
-    for (int i = 0; i < past.size(); ++i) {
-        if (!past.test(i)) {
-            all_landmarks_past = false;
-            break;
-        }
-    }
+    OperatorsProxy operators = task_proxy.get_operators();
 
     for (OperatorID op_id : applicable_operators) {
         OperatorProxy op = task_proxy.get_operators()[op_id];
```

Comment:
Since you already defined `operators` on line 128 I would use it: `operators[op_id]`.
Alternatively, remove the definition on line 128.